src/adapters/supabase/helpers/user.ts

```diff
@@ -0,0 +1,24 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Super } from "./supabase";
+import { Context } from "../../../types/context";
+
+type Wallet = {
+  address: string;
+};
+
+export class User extends Super {
+  constructor(supabase: SupabaseClient, context: Context) {
+    super(supabase, context);
+  }
+
+  async getWalletByUserId(userId: number, issueNumber: number) {
```

Comment:
What do we need this for?

Reply:
It is not used and it came with the template. Could be removed completely or used in TG slash commands whichever, likely remove since we are eradicating the DB.

Reply:
I suppose make a mental note of this and remove if we aren't using but maybe could be interesting in the future for deeper integrations with our system (we store user wallets) across GitHub and Telegram.